.devcontainer/post_start_command.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cd api && pip install -r requirements.txt
```

Comment:
Do you think we should add "flask db upgrade" at the end? I'm not using devcontainer, so I'm not very familiar with this practice.

Reply:
Is there a way to make a dry-run and just inform the user that the db is out of sync?
I think we should not overwrite the db just because the developer opens the devcontainer. By the way, I would do this in another PR anyway since I didn't add it, just moved it.